src/ai/backend/web/proxy.py

```diff
@@ -131,31 +131,31 @@ async def close_upstream(self) -> None:
 async def decrypt_payload(request: web.Request, handler) -> web.StreamResponse:
     request_headers = extra_config_headers.check(request.headers)
     secure_context = request_headers.get("X-BackendAI-Encoded", None)
-    if secure_context:
-        if not request.content:
-            request["payload"] = ""
-            return await handler(request)
-        config = request.app["config"]
-        scheme = config["service"]["force_endpoint_protocol"]
-        if scheme is None:
-            scheme = request.scheme
-        api_endpoint = f"{scheme}://{request.host}"
-        payload = await request.text()
-        initial_vector, real_payload = payload.split(":")
-        key = (base64.b64encode(api_endpoint.encode("ascii")).decode() + initial_vector * 2)[0:32]
-        crypt = AES.new(
-            bytes(key, encoding="utf8"), AES.MODE_CBC, bytes(initial_vector, encoding="utf8")
-        )
-        b64p = base64.b64decode(real_payload)
-        dec = unpad(crypt.decrypt(bytes(b64p)), 16)
-        result = dec.decode("UTF-8")
-        if not result:
-            request["payload"] = ""
+    try:
+        if secure_context:
+            if not request.content:
+                request["payload"] = ""
+                return await handler(request)
+            config = request.app["config"]
+            scheme = config["service"]["force_endpoint_protocol"]
+            if scheme is None:
+                scheme = request.scheme
+            api_endpoint = f"{scheme}://{request.host}"
+            payload = await request.text()
+            initial_vector, real_payload = payload.split(":")
+            key = (base64.b64encode(api_endpoint.encode("ascii")).decode() + initial_vector * 2)[
+                0:32
+            ]
+            crypt = AES.new(
+                bytes(key, encoding="utf8"), AES.MODE_CBC, bytes(initial_vector, encoding="utf8")
+            )
+            b64p = base64.b64decode(real_payload)
+            request["payload"] = unpad(crypt.decrypt(bytes(b64p)), 16)
         else:
-            request["payload"] = result
-    else:
-        request["payload"] = ""
-    return await handler(request)
+            request["payload"] = ""
+        return await handler(request)
+    except (Exception, base64.binascii.Error):
```

Comment:
```python
>>> import binascii
>>> binascii.Error.__mro__
(<class 'binascii.Error'>, <class 'ValueError'>, <class 'Exception'>, <class 'BaseException'>, <class 'object'>)
>>>
```
I think it would be best to just use `ValueError`.

Reply:
Also, even though `base64` internally refers `binascii`, it is not recommended to indirectly refer a module imported inside another module.  If you really want to use `binascii.Error`, just use `binascii.Error` instead of `base64.binascii.Error`.
In this case, the type error occurs because `binascii` is not included in `base64.__all__` which serves a similar role as `export` in Javascript.